Makefile

```
@@ -48,8 +49,11 @@ bin/patch_statements_date.py:
 		https://raw.githubusercontent.com/openfun/potsie/v$(POTSIE_RELEASE)/scripts/patch_statements_date.py \
 		-o bin/patch_statements_date.py
 
+clean:
+	if [ -d $(WARREN_DATA_DIR) ]; then rm -r $(WARREN_DATA_DIR); fi
+
```

Since this rule does not target a file, you should declare it as a `PHONY` one:

```suggestion
clean:
	if [ -d $(WARREN_DATA_DIR) ]; then rm -r $(WARREN_DATA_DIR); fi
.PHONY: clean
```